### ISSUE TITLE
Support installing docker-compose

### ISSUE THREAD

**Issue by carolynvs** [getcarina/dvm#114]
Would it be useful to have dvm also download the docker-compose client?

```bash
$ dvm install --compose 1.7.1
# downloads docker-compose to $DVM_DIR/bin/docker-compose/1.7.1/docker-compose

$dvm use --compose 1.7.1
# adds the above directory to the path
```

It would be really neat to have dvm know which version of compose goes with a docker version. So that if you `dvm use 1.10.1`, the right version of docker-compose is also added to your path. No clue if that information is readily available though...

**Comment by rgbkrk** [comment]
Docker compose is written in Python. I wouldn't go anywhere near that.

**Comment by carolynvs** [comment]
@rgbkrk  `docker-compose` is just a binary, distributed from their [GitHub releases](https://github.com/docker/compose/releases) page similar to the docker client. What am I missing?

**Comment by rgbkrk** [comment]
Whoa! That's changed up! It's still Python so they must be creating a bundled executable. Way cool!

**Comment by carolynvs** [comment]
@everett-toews @jamiehannaford  Have you used `docker-compose` enough to know about how to match up a `docker-compose` version with a `docker` version? Is it just something you need to know from reading release notes?

**Comment by everett-toews** [comment]
@carolynvs Let's hold off on this until after DockerCon.

**Comment by jeffwidman** [comment]
Please make this optional, as I'd rather manage my `docker-compose` under python virtualenv's...

**Comment by carolynvs** [comment]
@jeffwidman Yup! If we did this, the default behavior of dvm would remain the same, and the capability to download compose would be controlled with an extra flag.

**Comment by carolynvs** [comment]
It looks to me like `docker-compose` isn't going away(?), based on the demos from the DockerCon keynote. I don't have time to add this right now anyway, but we'll see things look when I resurface in a month. 😄

**Comment by lifeiscontent** [comment]
@carolynvs any updates on this?

**Comment by carolynvs** [comment]
Not really, no. If you’d like to submit a PR adding support for docker-compose too I’d be happy to review!
